molecule/requirements.yml

```diff
@@ -4,4 +4,6 @@ collections:
   - name: kubernetes.core
   - name: operator_sdk.util
   - name: community.docker
-  - name: awx.awx
+  - name: https://github.com/ansible/awx.git#/awx_collection/
```

Comment:
is this URL correct? A local git clone with it fails.

Reply:
Yep, ansible-galaxy collection install -r requirements.yml seems to be OK with it

Also, pulled it from their docs https://docs.ansible.com/ansible/latest/collections_guide/collections_installing.html#install-multiple-collections-with-a-requirements-file

```
collections:
  - name: https://github.com/organization/repo_name.git
    type: git
    version: devel
```